backend/app/controllers/location.controller.js

```diff
@@ -3,80 +3,80 @@ const Location = db.locations;
 const Op = db.Sequelize.Op;
 
 exports.create = (req, res) => {
-    // Validate request
-    if (!req.body.locationName) {
-      res.status(400).send({
-        message: "locationName can not be empty!"
-      });
-      return;
-    }
-  
-    // Create a Location
-    const location = {
-      locationName: req.body.locationName,
-      locationCategory : req.body.locationCategory,
-      locationDescription: req.body.locationDescription,
-      locationNumber: req.body.locationNumber,
-      locationWebsite: req.body.locationWebsite,
-      locationImgUrl: req.body.locationImgUrl,
-      locationLat: req.body.locationLat,
-      locationLong: req.body.locationLong,
-      locationProvince: req.body.locationProvince,
-      locationAvgRating: req.body.locationAvgRating,
-      locationSumReview: req.body.locationSumReview,
-      locationSumCheckIn: req.body.locationSumCheckIn,
-      createBy: req.body.createBy,
-      locationStatus: req.body.locationStatus
-    };
-  
-    // Save Location in the database
-    Location.create(location)
-      .then(data => {
-        res.status(201).send(data);
-      })
-      .catch(err => {
-        res.status(500).send({
-          message:
-            err.message || "Some error occurred while creating the Location."
-        });
-      });
+  // Validate request
+  if (!req.body.locationName) {
+    res.status(400).send({
+      message: "locationName can not be empty!",
+    });
+    return;
+  }
+
+  // Create a Location
+  const location = {
+    locationName: req.body.locationName,
+    locationCategory: req.body.locationCategory,
+    locationDescription: req.body.locationDescription,
+    locationNumber: req.body.locationNumber,
+    locationWebsite: req.body.locationWebsite,
+    locationImgUrl: req.body.locationImgUrl,
+    locationLat: req.body.locationLat,
+    locationLong: req.body.locationLong,
+    locationProvince: req.body.locationProvince,
+    locationAvgRating: req.body.locationAvgRating,
+    locationSumReview: req.body.locationSumReview,
+    locationSumCheckIn: req.body.locationSumCheckIn,
+    createBy: req.body.createBy,
+    locationStatus: req.body.locationStatus,
   };
```

Comment:
```suggestion
  const location = req.body
```
ตัว `location` ไม่เท่ากับ `req.body` หรอ?

Reply:
ทำไมต้องเขียนแบบนี้อะ มันก็เหมือนกันไม่ใช่อ่อ
@Goggags123

Reply:
งง ถ้าตรงคำว่า”มันก็เหมือนกันไม่ใช่อ่อ” อะไรเหมือนกัน??